template/assets/css/base/_grid.scss

```diff
@@ -166,11 +169,19 @@
 .sml-flex-col {
   flex-direction: column;
 }
+.sml-flex-col > * {
+  margin-right: 0;
+}
 .med-flex-row {
   @include respond-to(med) {
     flex-direction: row;
   }
 }
+.med-flex-row > * {
+  @include respond-to(med) {
+    margin-right: $gutter*2;
```

Comment:
This is an over-write. I'd like to do a bigger refactor of the flexbox grid options so that they're mobile first and more flexible in a future PR.